api/src/main/java/io/opencensus/trace/export/SampledSpanStore.java

```diff
@@ -48,6 +51,17 @@
 
   protected SampledSpanStore() {}
 
+  /**
+   * Returns a {@code SampledSpanStore} that maintains a map of span names, but always returns an
+   * empty list of {@link SpanData}.
+   *
+   * @return a {@code SampledSpanStore} that maintains a map of span names, but always returns an
```

Comment:
s/map/set ?

Reply:
Hi sebright, what does this comment mean? Should I use other libraries?

Reply:
I was wondering if the Javadoc should say "set" instead of "map" since the no-op class doesn't pair the span names with anything.

Reply:
I see. I agree that it should be updated.